### DISCUSSION TITLE
Git hub error message

### DISCUSSION THREAD

**Discussion by LJoyClifford** [community/community#23894] [New to GitHub]
<p>Hello! I would appreciate some help with solving this error.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/d/4dd8ceb554b462073512f724ef3a13bab92e89d9.png" data-download-href="/uploads/short-url/b6FgVCfB3vDHiHUSYg6921NTJ7X.png?dl=1" title="git hub error" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/78510843/181079870-e1f5790c-5953-4f7a-b08d-3a45487fdb1a.png" alt="git hub error" data-base62-sha1="b6FgVCfB3vDHiHUSYg6921NTJ7X" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4dd8ceb554b462073512f724ef3a13bab92e89d9_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4dd8ceb554b462073512f724ef3a13bab92e89d9_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4dd8ceb554b462073512f724ef3a13bab92e89d9_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4dd8ceb554b462073512f724ef3a13bab92e89d9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">git hub error</span><span class="informations">1920×1080 205 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>Looks like you’re trying to using Git over SSH to clone, but either you don’t have an SSH key pair for authentication set up on the machine, or you haven’t added it to your GitHub account. Please see: <a href="https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh/about-ssh" class="inline-onebox">About SSH - GitHub Docs</a></p>
